script/deinstall_clang.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+
+# deinstall clang because different clang OpenMP versions can not stay side by side
+clang_version=$(echo $CXX_COMPILER | tr -d "clang++-")
+echo "deinstall Clang-version: $clang_version"
+
+if ! agc-manager -e clang@${clang_version}; then
```

Comment:
The information of the `agc-manager` is "static". If you don't add the clang version to the `/etc/installed_software.json` file manually, the manager does not recognize, that you installed clang.

Reply:
We clarified it in an offline discussion. This is for the future, if clang is already installed in the container it does not try to uninstall it, because the user don't know, how it was installed.